scripts/acn/k8s/deployment.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ph-deployment-name-here
+  namespace: ph-deployment-namespace-here
+spec:
+  selector:
+    app: ph-deployment-name-here
+  ports:
+    - name: libp2p 
+      protocol: TCP
+      port: ph-node-port-number-here
+      targetPort: ph-node-port-number-here
+    - name: tcp
+      protocol: TCP
+      port: ph-node-delegate-port-number-here
+      targetPort: ph-node-delegate-port-number-here
+---
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Now the deployment is stateful and currently log file is appended across restarts